src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/CodeActions/CodeActionEndToEndTest.NetFx.cs

```diff
@@ -1132,6 +1141,126 @@ await ValidateExtractComponentCodeActionAsync(
             codeActionResolversCreator: CreateExtractComponentCodeActionResolver);
     }
 
+    [Fact]
+    public async Task Handle_ExtractComponent_NestedNodes()
+    {
+        var input = """
+            @page "/"
+            @namespace MarketApp.Pages.Product.Home
+
+            namespace MarketApp.Pages.Product.Home
+
+            <PageTitle>Home</PageTitle>
+
+            <div id="parent">
+                <div>
+                    <div>
+                        <div>
+                            <h1>[|selection:Div a title</h1>
```

Comment:
Is the `selection:` bit here intentional?

Reply:
nope, just copy and paste failure